packages/shared/src/components/cards/WelcomePostCard.tsx

```diff
@@ -26,6 +26,7 @@ export const WelcomePostCard = forwardRef(function SharePostCard(
     children,
     style,
     onReadArticleClick,
+    enableSourceHeader,
```

Comment:
I would default it to enableSourceHeader = false just to keep it boolean always.